micropython/senml/manifest.py

```diff
@@ -1,5 +1,7 @@
-metadata(version="0.1.0")
+metadata(
+    version="0.1.0", description="SenML serialisation for MicroPython.", pypi="micropython-senml"
```

Comment:
Would it be prettier to add a `,` at the end to force Black to put these all on separate lines?

Reply:
Done.